### PR TITLE
Added new Coin Link feature

### PR DESCRIPTION
**Pull request by sebasaenz**
This new feature is added in response to the issue #1  
Please let me know if any adjustment should be made.
Thanks!

### CODE REVIEW

**Comment by RallyCreatorCoins** [comment]
@rally-ray @pashneal Can one of you please review this PR to ensure that it satisifies the requirements of #1 ? Thanks :D

File: widget.js [hunk 1]
```diff
@@ -135,5 +136,67 @@ window.addEventListener("onEventReceived", function(o) {
                 }
             })
         }
+
+        let coinLinkHelpResponse = `To generate a custom coin link, type ${fieldData.coinLinkCommand} <CoinName> <COIN/USD> <Amount> <Memo>`;
```
Comment (Ju99ernaut) [inline widget.js]: Change `COIN` to `COINS`

File: widget.js [hunk 2]
```diff
@@ -135,5 +136,67 @@ window.addEventListener("onEventReceived", function(o) {
...
+                        if (4 == coinNameReq.readyState && 200 == coinNameReq.status) {
+                            let t = JSON.parse(coinNameReq.responseText);
+
+                            if (!t || t.length == 0) {
+                                errors.push("invalid coin");
+                            } else if (!t.filter(coin => coin.coinSymbol == coinLinkArgs[0].toUpperCase()).length) {
+                                errors.push("invalid coin");
+                            }
+
+                            if (["COIN", "USD"].indexOf(coinLinkArgs[1].toUpperCase()) == -1) {
```
Comment (Ju99ernaut) [inline widget.js]: Change `COIN` to `COINS`

**Review by Ju99ernaut — CHANGES_REQUESTED** [review]

File: widget.js (hunk 1 above)
Comment (sebasaenz) [reply widget.js]: Done!

File: widget.js (hunk 2 above)
Comment (sebasaenz) [reply widget.js]: Done!

**Review by Ju99ernaut — APPROVED** [review]

**Pull request merged**